### ISSUE TITLE
Add basic validation for inline configuration inputs

### ISSUE THREAD

**Issue by cjoudrey** [cjoudrey/graphql-schema-linter#238]
When specifying an invalid rule in an inline configuration, `graphql-schema-linter` should let the user know.

For example, this should raise an error:

```graphql
# lint-disable some-invalid-rule

type Query {
  field: String
}
```

Perhaps the process could terminate with an error like:

```
The inline configuration on line 1 of `filename.graphql` references a non-existing rule: some-invalid-rule
```

Similarly, we should gracefully handle when an inline configuration is specified without any rules.

This currently crashes:

```graphql
# lint-disable

type Query {
  field: String
}
```

We could either (1) make it raise an error like above or (2) support this as a feature and make it disable _all_ rules.

/cc @mshwery in case you're bored 😄
